client.go

```diff
@@ -41,6 +41,7 @@ type Client struct {
 	ircToken              string
 	connection            *tls.Conn
 	connActive            tAtomBool
+	onNewWhisper          func(channel string, user User, message Message)
```

Comment:
channel makes no sense for whispers aswell, so just remove it.

Reply:
Agreed. Good call.